src/locale/dim.json

```diff
@@ -415,7 +418,6 @@
   "LoadoutBuilder": {
     "AssumeMasterwork": "Assume Masterworked Stats",
     "AssumeMasterworkDetailed": "Generate sets assuming all items are masterworked.",
-    "BadPerk": "A perk that \"slightly\" modifies isn't the best option.",
```

Comment:
@delphiactual I seem to have i18n cleaning up a bunch of stuff. Do you want me to back these out?

Reply:
nope go ahead and let them through